crates/wysiwyg/src/dom/dom_struct.rs

```diff
@@ -72,6 +72,15 @@ where
         self.document().children()
     }
 
+    #[cfg(feature = "js")]
```

Comment:
If we remove this `#[cfg]`, when the `js` feature is turn off, the compiler will raise a warning (saying it's unused). I don't like warnings. If someone would like to use it for something else than `js`, this someone is free to remove the `#[cfg]`.

I've added this method to avoid having to clone the `&Vec<DomNode<S>>` returned by `children()`.